package.json

```diff
@@ -41,7 +41,7 @@
     "postcss": "^8.4.31",
     "prettier": "^3.0.3",
     "prismic-ts-codegen": "^0.1.19",
-    "slice-machine-ui": "^1.18.0",
+    "slice-machine-ui": "^2.10.5",
```

Comment:
Are these dep changes necessary? Any breaking changes for us with the major version bump?